src/styles/components/_override.dialog.scss

```diff
@@ -3,7 +3,7 @@
 
     // Replaces CMS icon with M.gov icon
     .ds-c-dialog__close {
-        background: url(./images/CloseSymbol.svg) center center no-repeat !important;
+        background: url(../../images/CloseSymbol.svg) center center no-repeat !important;
```

Comment:
@nichia, it looks like this variable would either not be available (not sure) or be interpolated to be different than what we think we need. The way the DS is currently set up, these paths appear to not be resolved during the build, and on our end (in MCT), they are resolved relative to where they are used, which ends up breaking things. We were hoping that using a relative path in the DS would solve this problem. 

Unfortunately, with our setup using Docker and Docker Compose, we cannot successfully test with a modified local version of the DS. I'm suspecting that a modification to the build scripts of the DS might be able to fix this issue for us, but we thought this change might be the most direct way to try to fix the problem right now.

Reference from `@cmsgov/design-system/packages/design-system/src/styles/settings/variables/_build.scss` (I'm not sure whether this is imported in mgov, because I don't have a local, built copy of the parent DS)

```scss
$image-path: '../images' !default;
```

Reference from `@cmsgov/mgov-design-system/docs/src` or `@cmsgov/mgov-design-system/examples/src` - looks like this wouldn't be available where you've cited

```scss
$image-path: './images' !default;
```